fineract-provider/src/test/java/org/apache/fineract/infrastructure/core/boot/tests/AbstractSpringBootWithMariaDB4jIntegrationTest.java

```diff
@@ -20,21 +20,19 @@
 
 import org.apache.fineract.ServerWithMariaDB4jApplication;
 import org.junit.runner.RunWith;
-import org.springframework.boot.test.IntegrationTest;
-import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 import org.springframework.test.context.web.WebAppConfiguration;
 
 @RunWith(SpringJUnit4ClassRunner.class)
-@SpringApplicationConfiguration(classes = ServerWithMariaDB4jApplication.Configuration.class)
+@SpringBootTest(classes = ServerWithMariaDB4jApplication.Configuration.class)
 @WebAppConfiguration
 @ActiveProfiles("basicauth")
-@IntegrationTest({ "server.port=0", "management.port=0", "mariaDB4j.port=0", "mariaDB4j.dataDir=null" })
```

Comment:
@ivan333m why remove this?

Reply:
@IntegrationTest was deprecated as of 1.4 in favor of org.springframework.boot.test.context.SpringBootTest.

Reply:
Oh OK, thanks.